### PR TITLE
added twilio call feature for code verification

### PR DESCRIPTION
**Pull request by Samy-33**
Changes:  
Added a method in `core.py` to send a call with verification code  
Made length of the verification code customizable by defining a variable `VERIFICATION_CODE_LENGTH`  
I couldn't test the code as the calling demo is not available in India. @MichaelYusko, If possible can you test the code?

### CODE REVIEW

**Comment by MichaelYusko** [comment]
@Samy-33 Hey! thanks for PR! I'll review since Saturday, because till Friday I don't have a free time :(

**Comment by Samy-33** [comment]
@MichaelYusko Cool, man :)

File: flask_notifyAll/core.py [hunk 1]
```diff
@@ -65,6 +65,30 @@ def send_verification_code(self, to_user, verification_code):
         )
         return verification_code
 
+    def make_notification_call(self, to_user, from_user, resource_url):
+        """
+        It is expected that resource_url contains already a text with random verification code
```
Comment (MichaelYusko) [inline flask_notifyAll/core.py]: @Samy-33 Could we put comments something like

```
def f():
  """"It is expected that resource...

```

File: flask_notifyAll/core.py [hunk 2]
```diff
@@ -65,6 +65,30 @@ def send_verification_code(self, to_user, verification_code):
...
+        TODO: Maybe automate putting verification code in the url
```
Comment (MichaelYusko) [inline flask_notifyAll/core.py]: @Samy-33 And remove this line

File: flask_notifyAll/constants.py [hunk 3]
```diff
@@ -0,0 +1 @@
+VERIFICATION_CODE_LENGTH = 6
```
Comment (MichaelYusko) [inline flask_notifyAll/constants.py]: @Samy-33 Also, I prefer to use 4 digits instead of 6.

**Pull request closed without merging**